keyboards/thepanduuh/degenpad/keymaps/via/keymap.c

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2023 ThePanduuh <thepanduuh.kb@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  /*Base*/
+  [0] = LAYOUT_numpad_6x4(
```

Comment:
I would highly recommend using the ortho layout, as it populates all of the spots and lets you properly set a default behavior for all of the keys (rather than some of them being "No"s)

Reply:
Makes sense, I have made this change.